httputil/httputil.go

```diff
@@ -84,6 +84,8 @@ func GetLogger(r *http.Request) logrus.FieldLogger {
 	return logging.NewMasterLogger()
 }
 
+// todo: investigate if it's used throughout the services (didn't work properly for UT)
```

Comment:
Seems like this is used by other services. What exactly does not work? I guess if the new structured logger works well, we might get rid of it.